.processes/wallets.md

```diff
@@ -0,0 +1,35 @@
+# Wallets
+
+At HOPR we create and use multiple Ethereum addresses that are under the control and supervision of the HOPR Association.
+
+Here are some of these wallets:
```

Comment:
Pls check in with Jakob if there's anything else to add beyond:
- main net Safe (we're transitioning there, it has)
- xDAI Wallet (legacy)
- xDAI Safe
- NFT minter(s) -> @QYuQianchen ?
- DAO Safe (make clear that this is NOT on the books or under the control of HOPR Association or HOPR Services AG)

HOPR Deployer: used to deploy production contracts and multi-sig signer
Multisig signers:
- 0x5AB4f2a41DEb3B925B23a3f7E00F206BED18ABB3
- 0x93bC372b4cC142dA75a365C5cB45be996347bfeC (only on Wallets)
- 0x50677B7e720102c5126e17f4485149208d3fce71 (only on Safes)

Reply:
AFAIK the NFT minter is `0xD7682Ef1180f5Fc496CF6981e4854738a57c593E`, you can see all its `batchMint` & `renounceRole` [transactions](https://blockscout.com/xdai/mainnet/address/0xD7682Ef1180f5Fc496CF6981e4854738a57c593E/transactions).

Confirmed with Jakob that these are the only ones, but I don't have the addresses. I think for now the `HOPR Deployer` and `HOPR Multisig` (i.e. `0x4f50ab4e931289344a57f2fe4bbd10546a6fdc17`) on top of the NFT minter one are enough for now.

Reply:
Solved in 5ff86c0